init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/service/JsonService.java

```diff
@@ -0,0 +1,55 @@
+package uk.ac.ox.ndph.mts.init_service.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.ac.ox.ndph.mts.init_service.config.GitRepo;
+import uk.ac.ox.ndph.mts.init_service.exception.InvalidConfigException;
+import uk.ac.ox.ndph.mts.init_service.model.Trial;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+
+@Service
+public class JsonService {
```

Comment:
not sure about the name.
can you also add description about what this class is

Reply:
TrialService? All methods either accept or return a Trial obj? Also possibly rename createTrialFromGitRepo to create(String fileName) ? Not sure if we need to add the location of the file in the method name, especially as there is only 1 method that creates

Reply:
Renamed to TrialService. Will rename once we replace this functionality with config-server, placed a TODO